ruby-lsp.gemspec

```
@@ -12,9 +12,7 @@ Gem::Specification.new do |s|
   s.homepage = "https://github.com/Shopify/ruby-lsp"
   s.license = "MIT"
 
-  s.files = Dir.chdir(File.expand_path(__dir__)) do
-    %x(git ls-files -z).split("\x0").reject { |f| f.match(%r{\A(?:test|spec|features|misc)/}) }
-  end
+  s.files = Dir.glob("{lib,exe}/**/*") + ["README.md", "CHANGELOG.md", "Gemfile", "VERSION", "LICENSE.txt"]
```

I don't think you need to pass the exe directory as it's already the binddir:

```suggestion
  s.files = Dir.glob("lib/**/*") + ["README.md", "CHANGELOG.md", "Gemfile", "VERSION", "LICENSE.txt"]
```

---

Also, do we want to include all kinds of files in `lib` or just `.rb` files?

---

I thought we needed to include the `exe` folder there (where the files have no extension). I added the `.rb` to the end of the pattern now (although we only have `rb` files in `lib` anyway).